### PR TITLE
Inline Input component

### PR DESCRIPTION
**Pull request by steppy452**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
Adds an inline input component which expands to your input

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

https://github.com/reaviz/reablocks/assets/1513140/9e5b1cc9-e1e1-4878-9f39-c21b3b4599d7

### CODE REVIEW

File: src/form/Input/InlineInput/InlineInput.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { FC, forwardRef, Ref, InputHTMLAttributes } from 'react';
+import AutosizeInput from 'react-input-autosize';
+import classNames from 'classnames';
+import { InputRef } from '../Input';
+import css from './InlineInput.module.css';
+
+export const InlineInput: FC<InputHTMLAttributes<HTMLInputElement>> =
+  forwardRef(({ className, ...rest }, ref: Ref<InputRef>) => (
+    <div
+      style={{
```
Comment (amcdnl) [inline src/form/Input/InlineInput/InlineInput.tsx:10]: lineline css ?

File: src/form/Input/InlineInput/InlineInput.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { FC, forwardRef, Ref, InputHTMLAttributes } from 'react';
+import AutosizeInput from 'react-input-autosize';
+import classNames from 'classnames';
+import { InputRef } from '../Input';
+import css from './InlineInput.module.css';
+
+export const InlineInput: FC<InputHTMLAttributes<HTMLInputElement>> =
```
Comment (amcdnl) [inline src/form/Input/InlineInput/InlineInput.tsx:7]: We should update select to use this.

**Review by amcdnl — CHANGES_REQUESTED** [review]

File: src/form/Input/InlineInput/InlineInput.tsx (hunk 1 above)
Comment (steppy452) [reply src/form/Input/InlineInput/InlineInput.tsx:10]: whoops thought I put that on the story

File: src/form/Select/SelectInput/SelectInput.tsx [hunk 3]
```diff
@@ -10,7 +10,7 @@ import React, {
 } from 'react';
 import classNames from 'classnames';
 import { SelectOptionProps, SelectValue } from '../SelectOption';
-import AutosizeInput from 'react-input-autosize';
```
Comment (amcdnl) [inline src/form/Select/SelectInput/SelectInput.tsx:13]: Can you remove this depedency now?

**Review by amcdnl — APPROVED** [review]

**Pull request merged**